frontends/web/src/components/guide/entry.tsx

```diff
@@ -57,6 +61,9 @@ export const Entry: FunctionComponent<TProps> = props => {
             {entry.link && (
               <p><A data-testid="link" href={entry.link.url}>{entry.link.text}</A></p>
             )}
+            {entry.link2 && (
+              <p><A data-testid="link" href={entry.link2.url}>{entry.link2.text}</A></p>
+            )}
```

Comment:
what if there are 5 more exchanges/brokers?

nicer IMO would be if the exchange/broker selection view would be generic (no link). and once the user selected a service the correct one would show up.

Reply:
Good point, now there is an optional parameter that allows linking the correct privacy policy, or no privacy policy at all if the exchange is not defined.